ScrapPackedExplorerCli/CliOptions.cs

```diff
@@ -62,14 +64,41 @@ class ExtractOptions : BaseOptions
     [Verb("list", HelpText = "list or search files and folders in the archive")]
     class ListOptions : BaseOptions
     {
-        [Option('l', "outputStyle", Required = false, Default = "list", HelpText = "Output list (default) or tree view")]
-        public string outputStyle { get; set; }
+        [Option('l', "outputStyle", Required = false, Default = OutputStyles.List, HelpText = "Output list (default) or tree view")]
+        public OutputStyles outputStyle { get; set; }
 
         [Option('q', "searchString", Required = false, Default = "", HelpText = "A Search string to filter the output with")]
         public string searchString { get; set; }
 
-        [Option('r', "regex", Required =false, Default = false, HelpText = "Defines if the search string is a regular expression")]
+        [Option('r', "regex", Required = false, Default = false, HelpText = "Defines if the search string is a regular expression")]
         public bool isRegex { get; set; }
+
+        // todo: come up with better description
+        // todo: change arguments style
+        [Option('s', "starts-with", Required = false, Default = false, HelpText = "Apply search query only to beginnng of the files path \nBy default applies everywhere")]
```

Comment:
as discussed on discord: maybe change to `--match-beginning` / `-b`

also maybe change `--show-file-size` → `-s` and `--show-file-offset` → `-o`

Reply:
also ok if only in next PR as that has to do with cli arguments

Reply:
`-o` is already for `--outputFile`

Reply:
not in list verb, right?
I don't care about that one though

Reply:
Oh, I forgot `list` don't have this. 
Got it